tests/test_mediaconnect/test_mediaconnect.py

```diff
@@ -209,13 +223,15 @@ def test_add_flow_vpc_interfaces_fails():
         client.add_flow_vpc_interfaces(FlowArn=flow_arn, VpcInterfaces=[])
     err = err.value.response["Error"]
     err["Code"].should.equal("NotFoundException")
-    err["Message"].should.equal("flow with arn=unknown-flow not found")
+    err["Message"].should.equal(
```

Comment:
This looks like a dodgy merge - this was changed as part of our flake8-upgrade, but this PR reverses it again

Reply:
woops, sorry, got myself a bit confused trying to get the testing/linting working locally. Thanks for pointing out the issue